src/Microsoft.SqlTools.ServiceLayer/ObjectExplorer/Nodes/TreeNode.cs

```diff
@@ -98,6 +98,11 @@ public string NodePathName
         /// </summary>
         public bool IsSystemObject { get; set; }
 
+        /// <summary>
+        // True if the node includes ledger objects
+        /// </summary>
+        public bool IsLedger { get; set; } = false;
```

Comment:
I'd rather we try to avoid having feature-specific flagsin a general class like TreeNode if we can. Most TreeNodes have nothing to do with Ledger stuff.

Reply:
I just pushed an update that removes the need for the IsLedger property by just checking for the node type id of DroppedLedgerColumns, is that a better solution?

Reply:
Much better, thank you 😄